src/main/java/net/fabricmc/loom/extension/MixinExtensionApiImpl.java

```diff
@@ -50,7 +50,7 @@ public abstract class MixinExtensionApiImpl implements MixinExtensionAPI {
 	public MixinExtensionApiImpl(Project project) {
 		this.project = Objects.requireNonNull(project);
 		this.useMixinAp = project.getObjects().property(Boolean.class)
-				.convention(project.provider(() -> !LoomGradleExtension.get(project).isNeoForge()));
+				.convention(project.provider(() -> !LoomGradleExtension.get(project).isNeoForge() && (!LoomGradleExtension.get(project).isForge() || !LoomGradleExtension.get(project).getForgeProvider().usesMojangAtRuntime())));
```

Comment:
Does the PR fix the crash bypassed by 8c2b72af727a813765e6082c95cfad6c2f903835?

Reply:
I never experienced any crash?
To get a crash here you would need to ref the property in your own buildscripts early, so imo just don't do it?

Reply:
So the crash in that commit is caused by architectury plugin, this should just require a change in arch plugin (which is in a separate branch there)
I think that is a better solution than making this false for forge.